having problem in building Hops

Hi,
When I  try to run " **mvn clean install generate-sources  package -Pdist,ndb,native -Dtar** ", I'm going to face below error. Does anyone has any comments or workaround about how I can fix this?

**[ERROR] Failed to execute goal on project hadoop-main: Could not resolve dependencies for project io.hops:hadoop-main:pom:2.8.2.7-SNAPSHOT: The following artifacts could not be resolved: io.hops.gpu:libhopsnvml:so:2.8.2.7-SNAPSHOT, io.hops.streaming:libhopsyarn:so:2.8.2.7-SNAPSHOT: Could not find artifact io.hops.gpu:libhopsnvml:so:2.8.2.7-SNAPSHOT in hops-snapshot-repository (https://bbc1.sics.se/archiva/repository/Hops/) -> [Help 1]
org.apache.maven.lifecycle.LifecycleExecutionException: Failed to execute goal on project hadoop-main: Could not resolve dependencies for project io.hops:hadoop-main:pom:2.8.2.7-SNAPSHOT: The following artifacts could not be resolved: io.hops.gpu:libhopsnvml:so:2.8.2.7-SNAPSHOT, io.hops.streaming:libhopsyarn:so:2.8.2.7-SNAPSHOT: Could not find artifact io.hops.gpu:libhopsnvml:so:2.8.2.7-SNAPSHOT in hops-snapshot-repository (https://bbc1.sics.se/archiva/repository/Hops/)**

--- a/docs/installation_guide/hops-manual-installation-guide.rst
+++ b/docs/installation_guide/hops-manual-installation-guide.rst
@@ -20,7 +20,8 @@ Download and Compile Sources
 ----------------------------
 Hops consists of two modules:
 
-* the Data Access Layer (DAL) and its' implementation (for a target database such as MySQL Cluster).
+* the Data Access Layer (DAL) and its implementation (for a target database such as MySQL Cluster).
+* the GPU abstraction layer and its implementation (for NVIDIA GPUs).
 * Hops Hadoop;
 
 Separating the data access layer permits different (distributed) transactional database storage engines with different licensing models. :ref:`More ... <hops-licensing>`
@@ -41,6 +42,21 @@ Download the source code for Data Access Layer Implementation for MySQL Cluster 
    > mvn install
 
 This generates a driver jar file ``./target/hops-metadata-dal-impl-ndb-1.0-SNAPSHOT.jar`` which is used by the HopsFS to communicate with the database. 
+
+Building the GPU Driver
+-------------------------------
+
+Download the source code for the GPU Layer Interface::
+
+  > git clone https://github.com/hopshadoop/hops-gpu-management.git
+  > cd hops-gpu-management
+  > mvn install -DskipTests
+
+Download the source code for the GPU layer Implementation for NVIDIA::
+
+  > git clone https://github.com/hopshadoop/hops-gpu-management-impl-nvidia.git
+  > cd hops-gpu-management-impl-nvidia
+  > mvn install -DskipTests
 
 Building Hops
 ----------------

